### DISCUSSION TITLE
Allow optios as autoplay/mute for video player when embeded

### DISCUSSION THREAD

**Discussion by dl-lim** [mediacms-io/mediacms#652] [Q&A]
Hi there, wonderful project here! :)

I can see that this can be embedded as an iframe, but are there currently also options to:
- loop
- autoplay
- turn off controls
- mute volume

Youtube currently does this with query string "?" after the URL and some other video embeds can be done in the `<video>` tag itself.

Wondering what is the way to do this for MediaCMS?

**Comment by mgogoulos** [comment]
Hi @alderson59 , this is not possible with the current embedded player but for sure they can be implemented. I'll slightly change this title to be more descriptive!

**Comment by mgogoulos** [comment]
Also setting a different colour per embed would be great

**Comment by EricGuic** [comment]
For embed purposes, and in addition to the `<iframe>` tag, is it possible to get an HTML5 `<video>` tag ?
Thanks.

**Comment by shodanx2** [comment]
Curious if autoplay/looping works in the player now ?
Also is it possible to animate thumbnail loops in the gallery view ?

**Comment by mgogoulos** [comment]
> Curious if autoplay/looping works in the player now ? Also is it possible to animate thumbnail loops in the gallery view ?

Give it a try on https://demo.mediacms.io/ !

**Comment by shodanx2** [comment]
Thanks, it seems there is not animation from the gallery view, at least not in the demo

I was thinking something like gfycat and maybe they even all play if you ask it

These kinds of short looping video are in a niche between photo galleries that support video and video sharing sites that expect videos being at least 10-20 seconds long at minimum

**Comment by shodanx2** [comment]
I have two mediacms instances running

It's great !

However I have to report currently autoplay, looping and gallery view animated preview are not functionning

And I have the encode profile preview/gif enabled in /admin/files/encodeprofile/ (it is on by default)
